christfellowshipchurch/web-app-v2
# Fixed BottomCard image for HeroListFeature

### About
The bottom card of the HeroList feature was not pulling the image in correctly and was not displaying any image at all.

### Test Instructions
* Go to HomeFeed and go to the "More Resources" card collection. You'll see the bottom image is now showing.

### Screenshots
|Before|After|
|----------|---------|
|![image](https://user-images.githubusercontent.com/46049974/134938886-7bb1b8b9-16fe-4599-81bc-e19769f400cc.png)|![image](https://user-images.githubusercontent.com/46049974/134938976-bbca7e50-641b-47e4-b01a-9e18957014c8.png)|

### Closes Tickets
[CFDP-1710]


[CFDP-1710]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1710
